### PR TITLE
Operate Public API documentation

### LINKED ISSUE #538: Add entry point for the public API documentation of Operate
Currently, technical documentation for the public API of Operate is available under` /swagger-ui.html` to read and try and a documentation in OpenAPI format is available under `/api-docs/v1`.

- [ ] Add an entry point to the public API of Operate in https://stage.docs.camunda.io/docs/apis-clients/overview/

### PR DESCRIPTION
**Pull request by ralfpuchert**
* Add Operate API documentation
   * Introduction, examples, and reference
* Add link for Operate API in  sidebar and public-api page

Closes #538, #513

### CODE REVIEW

File: docs/apis-clients/operate-api/index.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+#### Pagination
+
+Specify the item where the next search should start. For this, you need the values from previous results. 
+Copy the values from `sortValues` field from the previous results into the `searchAfter` value of query. 
+See also [results](#results).
+
+##### Example
+
+Get next page for: sorted by name descending:
```
Comment (christinaausley) [inline docs/apis-clients/operate-api/index.md]: What do you mean here?

File: docs/apis-clients/operate-api/index.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+
+#### Pagination
+
+Specify the item where the next search should start. For this, you need the values from previous results. 
+Copy the values from `sortValues` field from the previous results into the `searchAfter` value of query. 
+See also [results](#results).
+
+##### Example
+
+Get next page for sorted by name descending:
```
Comment (christinaausley) [inline docs/apis-clients/operate-api/index.md:140]: What do you mean here?

**Review by christinaausley — APPROVED** [review]
Technical adjustments made in my latest two commits, and one comment for clarity purposes. Latest commits included:
- Ensuring periods after each sentence.
- Adding spacing around headers for clean and easy reading code/Markdown purposes.
- Sentence case headers in tables as well.
- Flow node as opposed to flownode.

File: docs/apis-clients/operate-api/index.md (hunk 2 above)
Comment (ralfpuchert) [reply docs/apis-clients/operate-api/index.md]: Thanks, @christinaausley for the corrections.

I want to show the mechanism to get the next results for the previously described query. I changed the description in another commit. I hope it is now clearer.

**Pull request merged**